diff --git a/target/linux/qualcommax/Makefile b/target/linux/qualcommax/Makefile
index 8ffa7397c3..9954cd5c41 100644
--- a/target/linux/qualcommax/Makefile
+++ b/target/linux/qualcommax/Makefile
@@ -14,8 +14,9 @@ include $(INCLUDE_DIR)/target.mk
 DEFAULT_PACKAGES += \
 	kmod-usb3 kmod-usb-dwc3 kmod-usb-dwc3-qcom \
 	kmod-leds-gpio kmod-gpio-button-hotplug \
-	kmod-ath11k-ahb wpad-openssl \
-	uboot-envtools losetup \
-	automount cpufreq
+	kmod-ath11k kmod-ath11k-ahb wpad-openssl \
+	uboot-envtools e2fsprogs kmod-fs-ext4 losetup kmod-ramoops \
+	nss-tools luci-app-nss nssqos luci-app-nssqos \
+	sqm-scripts-nss luci-app-sqm
 
 $(eval $(call BuildTarget))
