diff --git a/feeds.conf.default b/feeds.conf.default
index 2a0d7f6c90..e3489b3a4c 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -1,4 +1,5 @@
-src-git packages https://github.com/immortalwrt/packages.git
+src-git packages https://github.com/solarflows/packages.git;qualcommax
+src-git nss https://github.com/JuliusBairaktaris/nss-packages.git;edma-nss
 src-git luci https://github.com/immortalwrt/luci.git
 src-git routing https://github.com/openwrt/routing.git
 src-git telephony https://github.com/openwrt/telephony.git
